kotlinpoet/src/main/java/com/squareup/kotlinpoet/LambdaTypeName.kt

```diff
@@ -80,12 +87,20 @@ public class LambdaTypeName private constructor(
   }
 
   public companion object {
+    /** Returns a lambda type with `returnType` and parameters listed in `parameters`. */
+    @DelicateKotlinPoetApi("Context receivers are currently a preview feature") @JvmStatic public fun get(
+      receiver: TypeName? = null,
+      contextReceivers: List<TypeName> = emptyList(),
```

Comment:
While I get why we can't just add this to the below one, let's at least add this argument to the end of the parameters list